docs/plugins/event-list.md

```diff
@@ -0,0 +1,55 @@
+---
+id: EventList
+title: List of Plugin Events
+sidebar_label: List of Plugin Events
+slug: /plugins/EventList
+---
+
+
+# List of Plugin Events
+
+- [Create Server](plugins\plugin-events\create-server.md)
+
+
+- [Create Package Json](plugins\plugin-events\create-package-json.md)
+
+
+- Resources 
+
+  - [Create Entity Service](plugins\plugin-events\create-entity-service.md)
+  - [Create Entity Service Base](plugins\plugin-events\create-entity-service-base.md)
+
+  - [Create Entity Controller](plugins\plugin-events\create-entity-controller.md)
+  - [Create Entity Controller Base](plugins\plugin-events\create-entity-controller-base.md)
+
+  - [Create Entity Resolver](docs\plugins\plugin-events\create-entity-resolver.md)
+  - [Create Entity Resolver Base](docs\plugins\plugin-events\create-entity-resolver.md)
+
+  - [Create Entity Controller Spec](docs\plugins\plugin-events\create-entity-controller-spec.md)
+
+
+- Create Entity DTO 
+
+- [Create Swagger](docs\plugins\plugin-events\create-swagger.md)
+
+- Create Server App Module
+
+- [Create Seed](docs\plugins\plugin-events\create-seed.md)
+
+- [Create Server Auth](docs\plugins\plugin-events\create-server-auth.md)
+
+- Create Entity DTO 
```

Comment:
You already mentioned it on line 31. 
Line 31 is the right place for create DTOs
```suggestion
```